reporters/reporter-perfrepo/src/main/java/org/radargun/reporting/perfrepo/PerfrepoReporter.java

```diff
@@ -91,7 +92,7 @@ public void validate() {
    }
 
    @Override
-   public void run(Collection<Report> reports) {
+   public void run(MasterConfig masterConfig, Collection<Report> reports) {
```

Comment:
Would it be possible to add this config to the ones that are uploaded to PerfRepo in the uploadAttachments method?